features/0023-connection-initiate/README.md

```diff
@@ -0,0 +1,425 @@
+# 0023: Connection Initiate Protocol
```

Comment:
Can we call this the "Initiate Connection" protocol instead of the "Connection Initiate" protocol. The latter sounds stilted.

Reply:
Even better, can we just call it the "connect" protocol?

Reply:
Let's let this simmer another few days.

Reply:
My name is terrible, but I don't have a good replacement yet.

Reply:
Another possibility is a name like "Hello" (if you didn't like connect)

Reply:
"Hello" is surprisingly elegant. It doesn't tick every box, but it's simple and meaningful.